source/docs/controls-overviews/control-system-hardware.rst

```diff
@@ -36,21 +36,21 @@ The goal of this document is to provide a brief overview of the hardware compone
 
         Diagram courtesy of FRC\ |reg| Team 3161 and Stefen Acepcion.
 
-## NI roboRIO
+## Limelight Systemcore
```

Comment:
Isn't it capitalised SystemCore?

Reply:
FIRST seems to use SystemCore, but all the Limelight docs use Systemcore.

Reply:
It looks like Limelight is moving to Systemcore: https://github.com/wpilibsuite/SystemCoreTesting/discussions/119#discussioncomment-13757158